src/oic/__init__.py

```diff
@@ -24,7 +24,7 @@
         )
 
 __author__ = "Roland Hedberg"
-__version__ = "1.2.0"
+__version__ = "1.3.0"
```

Comment:
Please do not update the version. It will be updated before the release in a separate commit.